src/app/inventory/store-types.ts

```diff
@@ -106,6 +107,10 @@ export interface DimStore {
     progressions: DestinyProgression[];
   };
 
+  uninstancedItemObjectives: null | {
```

Comment:
I don’t think you need to add these to the store just to pass them down to the function that makes items. Better to just pass this new map down as a new argument.